src/Components/Navbar/Navbar.jsx

```diff
@@ -1,5 +1,83 @@
+import React, { useState } from "react";
+// import { Link } from "react-router-dom";
+import { Icon } from "@iconify-icon/react";
+import styles from "./Navbar.module.scss";
+import logo from "./GDSC-Logo.png";
+
 const Navbar = () => {
-  return <nav>Navbar</nav>;
+  const [isActiveHome, setisActiveHome] = useState(true);
+  const [isActiveResources, setisActiveResources] = useState(false);
```

Comment:
Check the `NavLink` component provided by React Router v6 because if there would be 10 links in the Navbar, you would be making 10 states for managing those. It's a bad practice. 

Checkout [NavLink component by React Router v6](https://reactrouter.com/en/main/components/nav-link)